### PR TITLE
Add SC expansion protocol

### PR DESCRIPTION
**Pull request by fitzthum**
First draft of expansion procedure as discussed last week.

### CODE REVIEW

**Review by larrydewey — APPROVED** [review]

File: governance.md
Comment (dcmiddle) [inline governance.md:92]: I think we value code, docs, issues, and other artifacts that get committed to repos. Likewise maintainership implies substantive reviews of those artifacts (i.e. PR reviews). 
I'm not sure about participation in meetings though. The things that come to mind for me are things that lead to the aforementioned artifacts.

File: governance.md
Comment (magowan) [inline governance.md:96]: Chatting on with @larrydewey and @sameo , An idea to gently extend this is perhaps have a date at which the SC dissolve and reforms. We don't necessarily need to have elections at present as I don;t feel we have any issues regarding size of SC yet?
Perhaps our general principles about ensuring diverse representation from companies/views/contribution areas?
And our general principle to push contentious discussions into the weekly well attended meetings?

Comment (magowan) [reply governance.md:96]: But I am happy to approve what is here and we treat as a living document think about a further PR to add more.

Comment (magowan) [reply governance.md:96]: An opt in process when SC reforms to start with for both existing SC and wider community members, but SC is more admin/process longer term thoughts. But discussions decisions are driven by the weekly community calls.

**Review by magowan — APPROVED** [review]

**Comment by larrydewey** [comment]
It might also be good to have a cadence established for having say every n-th meeting dedicated for roadmap/architecture design.

**Comment by larrydewey** [comment]
Maybe add a section about encouraging and promoting members to present at specific events. Even reaching out to those individuals to present on what they are working on to make sure we have members from the community present at various CNCF events.

**Comment by larrydewey** [comment]
@sameo brought up a good idea about having members of the committee act as liaison between different organizations, such as the CNCF, to keep aware of events up-coming, and to also evangelize confidential containers during their meetings.

**Comment by fitzthum** [comment]
Ok, I have pushed a number of updates to this to incorporate comments, make certain things more explicit, and consider other types of membership changes besides expansion. PTAL.

File: governance.md
Comment (fidencio) [reply governance.md]: This has been solved in the last interaction, thanks for working on this @fitzthum!

**Comment by fitzthum** [comment]
Btw, it turns out that there is no provision in the charter for how to update the charter. We should probably add something like this later. For now, it seems reasonable to look for a voting quorum of TSC members to approve this PR and address any other comments that come up. Sound good?

**Comment by mkbhanda** [comment]
Odd, the links https://docs.google.com/document/d/15epr-ZDmlIp-8jR0HQAaK0aXNddYp-gdwZnvUFQysLw/edit?usp=sharing takes me to a placeholder page. My colleague, a non-TSC member has the same experience.

@fitzthum would you kindly fix the grammar in "
the latter being mostly focused on specific and technical details of one or more Confidential Containers project." with s/specific/specifics and s/project/sub-projects.

**Comment by mkbhanda** [comment]
Just became aware of @fidencio PR https://github.com/confidential-containers/confidential-containers/pull/149 to fix the link issue.

File: governance.md
Comment (sameo) [inline governance.md:91]:
```suggestion
Membership changes can fall into one of four categories, as described in the following sections.
```

File: governance.md
Comment (sameo) [inline governance.md:88]:
```suggestion
The convention of the SC is for each organization that is significantly engaged
in the project to be represented by one or two members. The term organization
covers any entity with a corporate, educational, or philantropic purpose like e.g.
companies, universities or foundations.
```

**Review by sameo — APPROVED** [review]
A couple of nits, otherwise LGTM.

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @fitzthum

**Comment by fitzthum** [comment]
Rebased to catch the updated link.

File: governance.md
Comment (mkbhanda) [reply governance.md:88]: tiny typo on line 89 - philanthropic

**Review by jiangliu — APPROVED** [review]

File: governance.md
Comment (mythi) [inline governance.md:95]: > Membership changes can be approved via GitHub and do not require an SC meeting.

does it mean a vote is same as a PR approval?

Comment (sameo) [reply governance.md:95]: Good call. I guess we should have a majority of the current TSC members approving the PR to get it merged?

Comment (mythi) [reply governance.md:95]: Agree, that was my thinking too.

**Review by peterzcst — APPROVED** [review]
LGTM

File: governance.md
Comment (fitzthum) [reply governance.md:99]: I clarified the voting procedure slightly. For expansion, 2/3rds are required. We don't have any means to reverse expansion decisions, so it's reasonable to have a higher bar. On GH that means 2/3rds of SC members approve the PR, which is roughly equivalent to a unanimous consensus at an SC meeting with a voting quorum (2/3rds) present.

Comment (fidencio) [reply governance.md:99]: > On GH that means 2/3rds of SC members approve the PR, which is roughly equivalent to a unanimous consensus at an SC meeting with a voting quorum (2/3rds) present.

IMHO, this is **very reasonable**, and more reasonable than a voting done in a meeting, as it gives time for the TSC members to jump in and approve (or not) the expansion, and also does **NOT** create any scenario where competitors could be blocked to join the project.

**Review by dcmiddle — APPROVED** [review]

**Review by sameo — APPROVED** [review]

**Review by jiazhang0 — APPROVED** [review]

File: governance.md
Comment (mythi) [inline governance.md:142]: not critical to add but maybe something to think about. in CNCF circles I've seen two types of votes: binding and non-binding where the latter is used by the community members to show signs of support. maybe this could be mentioned in the commit message at least that non-binding votes are also encouraged?

File: governance.md
Comment (c3d) [inline governance.md:104]: At the moment, big users of the project are also contributors. I wonder if at some point we will welcome non-contributing users (the definition of contributors being "submitting PRs"). If a company starts using CC a lot, and provides valuable feedback in the form of issues, would that be enough for us to consider this a "contribution"?

Comment (fidencio) [reply governance.md:104]: Speaking out of my mind here, based on my interpretation of the text, so bear with me. :-)

Your point comes from "(the definition of contributors being "submitting PRs").", but I do believe this is clarified a few lines down below, by the following sentence:
"There is no standard definition for significant contributions."

So, I do believe this will get into the "case by case analisys", but there's nothing in the text that would prevent those from being considered.

**Review by fidencio — APPROVED** [review]

File: governance.md
Comment (fitzthum) [reply governance.md:104]: Yes, contributions does not necessarily imply submitting PRs.

**Comment by fidencio** [comment]
@fitzthum, thanks for working on this, very much appreciated, I've linked #144 to this one, so it gets automatically closed once this one gets merged.

**Comment by sameo** [comment]
It seems we have a large number of approvals here, let's merge that one. We can follow up with adjustments.

**Pull request merged**
